contracts/ens/ens.go

```diff
@@ -101,6 +101,10 @@ func ensNode(name string) common.Hash {
 	return crypto.Keccak256Hash(parentNode[:], parentLabel[:])
 }
 
+func EnsNode(name string) common.Hash {
```

Comment:
This looks like an ugly hack. Maybe we should add a TODO to refactor the code and export `ensNode` or?

Reply:
I thought adding a public accessor like this was perfectly valid? It's done several other places in the `go-ethereum` code. But yes, you're right, we should open a PR asking for export in one way or another. 

Could also code one from scratch, of course, that would make things lighter even. But I can't help but feel that this should be possible to do through this package.

There is another library that offers a standalone tool, but it also includes a bunch of other code which would just be a waste of space in this case.

Reply:
Sure, it is valid, but why not just export `ensNode`? I can't think of a reason why one would want to have both `ensNode` and `EnsNode` in the same codebase, apart from the fact that this will take attention and focus away from your PR to the `ensNode` refactor - because of this, I think it is fine for now, but maybe just add a comment to consider refactor in the future.